wdfn-server/config.py

```diff
@@ -101,8 +91,18 @@
 
 # Mail settings for feedback form
 MAIL_SERVER = 'smtp.usgs.gov'
-EMAIL_TARGET = {
-    'contact' : 'gs-w-{state_district_code}_NWISWeb_Data_Inquiries@usgs.gov',
+EMAIL_TARGETS = {
+    'contact': 'gs-w-{state_district_code}_NWISWeb_Data_Inquiries@usgs.gov',
     'report': 'gs-w_support_nwisweb@usgs.gov',
     'comment': 'WDFN@usgs.gov'
 }
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
+LOOKUP_ENDPOINTS = {
+    'HUC_LOOKUP': 'https://labs-test-website.s3-us-west-2.amazonaws.com/test-lookups/huc_lookup.json',
+    'COUNTRY_STATE_COUNTY_LOOKUP': 'https://labs-test-website.s3-us-west-2.amazonaws.com/test-lookups/country_state_county_lookup.json',
+    'NWIS_CODE_LOOKUP': 'https://labs-test-website.s3-us-west-2.amazonaws.com/test-lookups/nwis_code_lookup.json'
+}
+
+# Use a comma separated string, with no spaces
+EMAIL_LIST_DEVELOPERS = 'abriggs@contractor.usgs.gov'
```

Comment:
Should probably list two developers eventually. Also wondering if we should make this an environment variable so that it can be set per tier.